tests/slow/src/test/scala/tests/sbt/SbtBloopLspSuite.scala

```diff
@@ -711,16 +711,16 @@ class SbtBloopLspSuite
 
   test("semantic-highlight") {
     val expected =
-      s"""|<<lazy>>/*modifier*/ <<val>>/*keyword*/ <<root>>/*variable,readonly*/ = (<<project>>/*class*/ <<in>>/*method*/ <<file>>/*method*/(<<".">>/*string*/))
+      s"""|<<lazy>>/*modifier*/ <<val>>/*keyword*/ <<root>>/*variable,readonly*/ = (<<project>>/*method*/ <<in>>/*method*/ <<file>>/*method*/(<<".">>/*string*/))
           |  .<<configs>>/*method*/(<<IntegrationTest>>/*variable,readonly*/)
           |  .<<settings>>/*method*/(
           |    <<Defaults>>/*class*/.<<itSettings>>/*variable,readonly*/,
           |    <<inThisBuild>>/*method*/(
-          |      <<List>>/*namespace*/(
```

Comment:
It was incorrect previously, because of multiple symbols at the same position.

Reply:
Actually, I've noticed the bug just today, good catch!